org.thecrossingchurch.CustomJobs/Jobs/HubspotIntegrationPatching.cs

```diff
@@ -0,0 +1,442 @@
+            Debug.WriteLine( string.Format( "Comparing {0} Top-Level Domains.", tlds.Length ) );
+
+            // ENews from DataView To List
+            var qry = enewsDV.GetQuery();
+            var eNewsData = qry.Select( item => item ).ToList();
+            HashSet<string> eNewsEmails = new HashSet<string>();
+            //string eNewsEmailList = "";
+            foreach ( var row in eNewsData )
+            {
+                string colVal = row.GetPropertyValue( "Email" ).ToStringSafe().ToLower();
+                if ( colVal != "" )
+                {
+                    eNewsEmails.Add( colVal );
+                    //eNewsEmailList = eNewsEmailList + "," + colVal;
+                }
+            }
+            Debug.WriteLine( string.Format("{0} Emails on ENews List.",eNewsData.Count ) );
+
+
+            HashSet<string> hsPersonEmails = new HashSet<string>();
```

Comment:
`HashSet`s aren't meant for this use case. After a quick test, I saw that a `List` is over two times faster for this use case. Can we swap this out for a `List`?

Reply:
Appears this is used for .contains() also